scripts/build_wheel.py

```diff
@@ -53,6 +53,8 @@
 https://github.com/python/typeshed/tree/master/stubs/{distribution}. All fixes for
 types and metadata should be contributed there.
 
+{extra_description}
```

Comment:
Maybe only add the extra newlines if the extra description is non-empty?

Reply:
I think that multiple empty lines should result in only one new paragraph in markdown, but I'm not certain that's true and whether all markdown renderers do that.

Should I just split the constant and use string concat?

Reply:
How you did with having `parts` looks good.